domains/aozora.json

```diff
@@ -6,6 +6,6 @@
     "email": "aozoradeveloper@gmail.com"
   },
   "record": {
-    "CNAME": "aozoradev.github.io"
+    "CNAME": "node3.foxxy.id:1837"
```

Comment:
Port number cannot be part of the CNAME record. You can create a CNAME for `node3.foxxy.id` and then access it as `aozora.is-a.dev:1837`. Alternatively you can create a URL redirection from `https://aozora.is-a.dev` to `http://node3.foxxy.id:1837`